src/core/diffusion_grid.h

```diff
@@ -516,7 +521,8 @@ class DiffusionGrid {
             l[2] = 0;
           } else {
             b = c - nx * ny;
-          }
+          }  // 2nd Order Runge-Kutta diffusion, often refered to as the
+             // improved Euler or midpoint method.
```

Comment:
Remove misplace comment. Put this on top of the RK method. This should not be here.